.grabl/automation.yml

```diff
@@ -0,0 +1,136 @@
+        ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME \
+        ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD \
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+        bazel build //...
+    test-concept:
+      machine: graknlabs-ubuntu-20.04
+      type: foreground
+      script: |
+        pyenv global 3.6.10
+        sudo unlink /usr/bin/python3
+        sudo ln -s $(which python3) /usr/bin/python3
+        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
+        ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME \
+        ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD \
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+        bazel test //:test_concept --test_output=errors
+    test-keyspace:
+      machine: graknlabs-ubuntu-20.04
+      type: foreground
+      script: |
+        pyenv global 3.6.10
+        sudo unlink /usr/bin/python3
+        sudo ln -s $(which python3) /usr/bin/python3
+        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
+        ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME \
+        ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD \
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+        bazel test //:test_keyspace --test_output=errors
+    test-answer:
+      machine: graknlabs-ubuntu-20.04
+      type: foreground
+      script: |
+        pyenv global 3.6.10
+        sudo unlink /usr/bin/python3
+        sudo ln -s $(which python3) /usr/bin/python3
+        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
+        ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME \
+        ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD \
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+        bazel test //:test_answer --test_output=errors
+    deploy-pip-snapshot:
+      filter:
+        owner: graknlabs
+        branch: master
+      machine: graknlabs-ubuntu-20.04
+      type: foreground
+      script: |
+        pyenv global 3.6.10
+        sudo unlink /usr/bin/python3
+        sudo ln -s $(which python3) /usr/bin/python3
+        sudo ln -s /usr/share/pyshared/lsb_release.py /opt/pyenv/versions/3.6.10/lib/python3.6/site-packages/lsb_release.py
+        ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME \
+        ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD \
+        bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
+        export DEPLOY_PIP_USERNAME=$REPO_GRAKN_USERNAME
+        export DEPLOY_PIP_PASSWORD=$REPO_GRAKN_PASSWORD
+        bazel run --define version=$(git rev-parse HEAD) //:deploy-pip -- snapshot
+    test-deployment:
+      filter:
+        owner: graknlabs
+        branch: master
+      machine: graknlabs-ubuntu-20.04
+      type: foreground
+      dependencies: [deploy-pip-snapshot]
+      script: |
+        pyenv global 3.6.10
```

Comment:
do we still need this @vmax ?

Reply:
@flyingsilverfin yes; it wouldn't build on Python 3.8 due to old `grpcio`

Reply:
right ok!